### PR TITLE
fix: fix dead link to UMA Optimistic Oracle

### PR DESCRIPTION
**Pull request by mrice32**
## Description

The link to UMA's Optimistic Oracle page is dead. This updates the link and fixes a minor typo.

## Related Issue

N/A

### CODE REVIEW

**Comment by ferhter** [comment]
0

**Comment by ferhter** [comment]
f3n4nd0m3-main

**Comment by ferhter** [comment]
0x0000000000000000000000000000000000

**Comment by ferhter** [comment]
[``](url)

File: src/content/developers/docs/oracles/index.md [hunk 1]
```diff
@@ -392,7 +392,7 @@ There are multiple oracle applications you can integrate into your Ethereum dapp
 
 **[Witnet](https://witnet.io/)** - _Witnet is a permissionless, decentralized, and censorship-resistant oracle helping smart contracts to react to real world events with strong crypto-economic guarantees._
 
-**[UMA Oracle](https://umaproject.org/products/optimistic-oracle)** - _UMA's optimistic oracle allows smart contracts to quickly and receive any kind of data for different applications, including insurance, financial derivatives, and prediction market._
+**[UMA Oracle](https://uma.xyz/#builder)** - _UMA's optimistic oracle allows smart contracts to quickly and receive any kind of data for different applications, including insurance, financial derivatives, and prediction markets._
```
Comment (corwintines) [inline src/content/developers/docs/oracles/index.md:395]:
```suggestion
**[UMA Oracle](https://uma.xyz)** - _UMA's optimistic oracle allows smart contracts to quickly and receive any kind of data for different applications, including insurance, financial derivatives, and prediction markets._
```

Just going to remove the id to match similar to what was there before: https://web.archive.org/web/20221228222831/https://umaproject.org/products/optimistic-oracle

**Review by corwintines — APPROVED** [review]

**Pull request merged**